docs/maui/converters/not-equal-converter.md

```diff
@@ -0,0 +1,93 @@
+---
+title: .NET MAUI NotEqualConverter
+author: bijington
+description: "The NotEqualConverter is a one way converter that returns a `bool` indicating whether the binding value is not equal to another specified value."
+ms.date: 03/02/2022
+---
+
+# NotEqualConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `NotEqualConverter` is a one way converter that returns a `bool` indicating whether the binding value is not equal to another specified value.
+
+The `Convert` method returns `true` when the binding `value` is **not equal** to the supplied `ConverterParameter`.
+
+The `ConvertBack` method is not supported. For the opposite behavior see the [`EqualConverter`](equal-converter.md).
+
+## Syntax
+
+### XAML
+
+The `NotEqualConverter` can be used as follows in XAML:
+
+```xaml
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+             x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.NotEqualConverterPage">
+
+    <ContentPage.Resources>
+        <ResourceDictionary>
+            <toolkit:NotEqualConverter x:Key="NotEqualConverter" />
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <Label Text="The value is not equal to 100"
+           IsVisible="{Binding MyValue, Converter={StaticResource NotEqualConverter}, ConverterParameter=100}" />
+
+</ContentPage>
+```
+
+### C#
+
+The `NotEqualConverter` can be used as follows in C#:
+
+```csharp
+
+class NotEqualConverterPage : ContentPage
+{
+    public NotEqualConverterPage()
+    {
+        var label = new Label { Text = "The value is not equal to 100" };
+
+		label.SetBinding(
+			Label.IsVisibleProperty,
+			new Binding(
+				nameof(ViewModels.MyValue),
+				converter: new EqualConverter(),
+				converterParameter: 100));
+
+		Content = label;
```

Comment:
@brminnick how does this render on the docs page? The indentation looks fine in VS Code on my machine but I wonder if I need to tweak the settings